clang/lib/Format/FormatToken.cpp

```diff
@@ -33,6 +33,18 @@ const char *getTokenTypeName(TokenType Type) {
   return nullptr;
 }
 
+static SmallVector<StringRef> QtPropertyKeywords = {
```

Comment:
```suggestion
static constexpr std::array<StringRef> QtPropertyKeywords = {
```
This can be constructed at compile time.